governance-exercises/StableLab-Poll-02/README.md

```diff
@@ -0,0 +1,35 @@
+# StableLab Poll Excercise 02
+
+## My Course of Action as a Governance Facilitator:
```

Comment:
As a general rule, pushing for more detail and specificity around how to deal with this situation being included in an edit to the governance scope is also a good idea.